amr-wind/wind_energy/actuator/wing/fixed_wing_ops.H

```diff
@@ -85,17 +116,53 @@ struct ReadInputsOp<FixedWing, ActSrcLine>
             amrex::max(max_eps, max_epsc) * max_chord * 3.0;
         const auto& p1 = wdata.start;
         const auto& p2 = wdata.end;
+        // Bounding box quantities used for all motion types
+        amrex::Real minpx = amrex::min(p1.x(), p2.x());
+        amrex::Real maxpx = amrex::max(p1.x(), p2.x());
+        amrex::Real minpy = amrex::min(p1.y(), p2.y());
+        amrex::Real maxpy = amrex::max(p1.y(), p2.y());
+        amrex::Real minpz = amrex::min(p1.z(), p2.z());
+        amrex::Real maxpz = amrex::max(p1.z(), p2.z());
+        // Bounding box limits for 2D actuator
+        if (wdata.gauss_2D) {
+            // Assume largest component of span needs entire domain, but the
+            // others do not need the bounding box to grow
+            const auto wspan = wdata.end - wdata.start;
+            // Flag for checking periodicity in spanwise direction
+            int warn_per = -1;
```

Comment:
feels like this should be a bool since we don't care what the value is. And then the checks below become much simpler.

Reply:
done. I meant to use this to change the warning depending on the direction but it was a bad approach anyway